cmd/kubeadm/app/cmd/upgrade/common_test.go

```diff
@@ -19,50 +19,100 @@ package upgrade
 import (
 	"bytes"
 	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
 	"testing"
 
+	clientset "k8s.io/client-go/kubernetes"
```

Comment:
```suggestion
	clientset "k8s.io/client-go/kubernetes"

```

import groups are

stdlib

external

k8s.io misc

kubeadm

Reply:
Gocha, will take some chance to clean this up in the future.